omod/src/main/java/org/openmrs/module/bedmanagement/rest/resource/AdmissionLocationResource.java

```diff
@@ -58,7 +58,7 @@
 
 @Resource(name = RestConstants.VERSION_1
         + "/admissionLocation", supportedClass = AdmissionLocation.class, supportedOpenmrsVersions = { "1.9.*", "1.10.*",
-                "1.11.*", "1.12.*", "2.0.*", "2.1.*" })
+                "1.11.*", "1.12.*", "2.0.*", "2.1.*", "2.2.*", "2.3.*", "2.4.*" })
```

Comment:
You may wanna just do `1.11.* - 9.*`

Reply:
Hi Daniel,
As of now we have just upgraded to 2.4.2, that's why only kept till 2.4.*
So should just make a commit of 1.11.* - 2.4.*?

Reply:
Do you intend to keep adjusting this every time you upgrade? 
The `-9.*` protects you from having to do that, because most of the times you do not have to change anything apart from just bumping the version. Which has turned out to be painfully unnecessary.

Reply:
@dkayiwa, 
yes, got your point.
Updated version range to * - 9.*
please review once